connector/connect/src/test/scala/org/apache/spark/sql/connect/planner/SparkConnectProtoSuite.scala

```diff
@@ -141,10 +149,35 @@ class SparkConnectProtoSuite extends PlanTest with SparkConnectPlanTest {
     comparePlans(connectPlan2, sparkPlan2)
   }
 
-  test("column alias") {
+  test("[SPARK-40809] column alias") {
```

Comment:
nit but:
```suggestion
  test("SPARK-40809: column alias") {
```
per https://spark.apache.org/contributing.html